### PR TITLE
move scheduler policy to pkg

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
move scheduler policy to pkg, scheduler policy should be a public pkg
#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

### CODE REVIEW

**Review by Muzry — DISMISSED** [review]

**Review by Muzry — COMMENTED** [review]
lgtm

**Review by Muzry — COMMENTED** [review]
lgtm

**Comment by Muzry** [comment]
remove scheduler policy to pkg -> move scheduler policy to pkg

File: apistructs/policy.go [hunk 1]
```diff
@@ -101,6 +101,25 @@ func (c *ExecutorWholeConfigs) OrgOpt(org string) *OrgOpt {
 	return nil
 }
 
+// Org organization, Corresponding tenant concept
+type Org struct {
```
Comment (sfwn) [inline apistructs/policy.go:105]: Add Executor prefix.

Comment (sfwn) [reply apistructs/policy.go:105]: And other fields.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**